src/System.Net.Sockets/tests/FunctionalTests/SendFile.cs

```diff
@@ -101,7 +101,6 @@ public void NotConnected_ThrowsException()
         public void SendFile_Synchronous(IPAddress listenAt, bool sendPreAndPostBuffers, int bytesToSend)
         {
             const int ListenBacklog = 1;
-            const int LingerTime = 10;
```

Comment:
I'd prefer you make all related changes in a single PR.
We need to ensure that `LingerOptions` is tested somewhere else or write new tests before removing it from these tests.